commit ca799bc50411fe5a0cf79e60e97a2c3a0a2afdda
Author: mjw <mjw@a5019735-40e9-0310-863c-91ae7b9d1cf9>
Date:   Tue Jun 2 18:17:44 2015 +0000

    Enable process_vm_readv/writev syscall on arm32.
    
    This makes none/tests/process_vm_readv_writev.vgtest pass on arm32.
    
    git-svn-id: svn://svn.valgrind.org/valgrind/trunk@15304 a5019735-40e9-0310-863c-91ae7b9d1cf9

diff --git a/coregrind/m_syswrap/syswrap-arm-linux.c b/coregrind/m_syswrap/syswrap-arm-linux.c
index f8f56f5..32c41ca 100644
--- a/coregrind/m_syswrap/syswrap-arm-linux.c
+++ b/coregrind/m_syswrap/syswrap-arm-linux.c
@@ -1214,6 +1214,10 @@ static SyscallTableEntry syscall_main_table[] = {
    LINXY(__NR_clock_adjtime,     sys_clock_adjtime),    // 372
    LINX_(__NR_syncfs,            sys_syncfs),           // 373
    LINXY(__NR_sendmmsg,          sys_sendmmsg),         // 374
+
+   LINXY(__NR_process_vm_readv,  sys_process_vm_readv), // 376
+   LINX_(__NR_process_vm_writev, sys_process_vm_writev),// 377
+
    LINXY(__NR_getrandom,         sys_getrandom),        // 384
    LINXY(__NR_memfd_create,      sys_memfd_create)      // 385
 };
